scripts/eosio_build_centos.sh

```diff
@@ -176,18 +176,23 @@
 		printf "\tInstalling CMAKE\n"
 		mkdir -p ${HOME}/opt/ 2>/dev/null
 		cd ${HOME}/opt
-		curl -L -O https://cmake.org/files/v3.10/cmake-3.10.2.tar.gz
+		STATUS=$(curl -LO -w '%{http_code}' --connect-timeout 30 https://cmake.org/files/v3.10/cmake-3.10.2.tar.gz)
+		if [ "${STATUS}" -ne 200 ]; then
+			printf "\tUnable to download Boost libraries at this time.\n"
+			printf "\tExiting now.\n\n"
+			exit;
+		fi
 		tar xf cmake-3.10.2.tar.gz
 		rm -f cmake-3.10.2.tar.gz
-		ln -s cmake-3.10.2/ cmake
+		ln -s ${HOME}/opt/cmake-3.10.2/ cmake
 		cd cmake
 		./bootstrap
 		if [ $? -ne 0 ]; then
 			printf "\tError running bootstrap for CMAKE.\n"
 			printf "\tExiting now.\n\n"
 			exit;
 		fi
-		make
+		make -j${CPU_CORE}
```

Comment:
min((RAMinGB/2GB), CPU_CORE) would be better but not sure how easy that is to calculate.

Reply:
Do you mean something like:
	MEM_GIG=$(( (($MEM_MEG / 1000) / 2) ))
	JOBS=$(( ${MEM_GIG} > ${CPU_CORE} ? ${CPU_CORE} : ${MEM_GIG} ))

Reply:
Sure if that works everywhere.

Reply:
Implemented on CentOS and running a full dependency build now. Ill push it when I have a successful build.